gunicorn/workers/gthread.py

```diff
@@ -49,7 +49,8 @@ def init(self):
         if self.parser is None:
             # wrap the socket if needed
             if self.cfg.is_ssl:
-                self.sock = ssl.wrap_socket(self.sock, server_side=True,
+                context = ssl.create_default_context()
```

Comment:
this should be a separate commit and not related to pylint really as it is using new code.